### ISSUE TITLE
How to plot graph from activity.txt file.

### ISSUE THREAD

**Issue by bugcy013** [astrofrog/psrecord#39]
Hi Guys,

It's possible can we generate the graph from an activity.txt file. 

Please guide me How to do that.

Thankyou
-Dhanasekaran.

**Comment by astrofrog** [comment]
@saimn implemented a nice tool to do this here: https://github.com/astrofrog/psrecord/pull/9 - I'll try and review this soon as I agree it would be useful!

**Comment by graysky2** [comment]
I agree, replotting an existing log would be great particularly if it's possible to include the smooth command from https://github.com/astrofrog/psrecord/pull/49

**Comment by ricardojlrufino** [comment]
I extracted @graysky2 code to file ..
https://gist.github.com/ricardojlrufino/c0d485f7968dfdb58519fb24c97de5c9
Using: 
> python3 plotutil.py activity-home.txt -o plot.png


It would be very useful if the merge was done ..
